src/components/Section.js

```diff
@@ -56,7 +56,8 @@ class Section extends Component {
         paddingTop: mjAttribute('padding-top'),
         paddingBottom: mjAttribute('padding-bottom'),
         paddingRight: mjAttribute('padding-right'),
-        paddingLeft: mjAttribute('padding-left')
+        paddingLeft: mjAttribute('padding-left'),
+        padding: mjAttribute('padding')
```

Comment:
This should be on top of every paddingLeft/Right/Top/Bottom. They must override `padding` if they're present like so :
```xml
<mj-column padding="20 0" padding-left="40px">
```